includes/class.llms.ajax.handler.php

```diff
@@ -712,6 +712,14 @@ public static function select2_query_posts() {
 		}
 		$post_types = implode( ',', $post_types_array );
 
+		$post_statuses = sanitize_text_field( $_REQUEST['post_statuses'] );
```

Comment:
add test cases to test at https://github.com/gocodebox/lifterlms/commit/a24e61fd210a50354d778da6a81799caa93aa134 to test post status data added via this update.

Reply:
So I guess you want me to merge the master into this branch and implement the test (this will also give me the opportunity to fix the conflict generated by the doc additions in my latest commit), is it right?!

Reply:
tests added in a791556
let me know if you think we need some other tests.